services/node_get_info.proto

```diff
@@ -0,0 +1,132 @@
+syntax = "proto3";
+
+package proto;
+
+/*-
+ * ‌
+ * Hedera Network Services Protobuf
+ * ​
+ * Copyright (C) 2018 - 2021 Hedera Hashgraph, LLC
+ * ​
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * ‍
+ */
+
+option java_package = "com.hederahashgraph.api.proto.java";
+// <<<pbj.java_package = "com.hedera.hapi.node.node">>> This comment is special code for setting PBJ Compiler java package
+option java_multiple_files = true;
+
+import "basic_types.proto";
+import "custom_fees.proto";
+import "query_header.proto";
+import "response_header.proto";
+
+/**
+ * Gets information about Node instance
+ */
+message NodeGetInfoQuery {
```

Comment:
Updated.

Reply:
Question.  We're requiring super user just to query node information?

Reply:
yes

Reply:
Thats because we won't be able to test without this query in services and since the node transactions are privileged, we discussed in a meeting we need this query.